spec/controllers/admin_stats_controller_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'spec_helper'
+
+describe Admin::StatsController, type: :controller do
+  let(:user1) { FactoryGirl.find_or_create(:user) }
+  let(:user2) { FactoryGirl.find_or_create(:archivist) }
+  
+  before do
+    allow(user1).to receive(:groups).and_return(['umg/up.dlt.scholarsphere-admin-viewers'])
```

Comment:
Should the groups returned here be `['admin']` to match up the the check in the `StatsAdmin` class, rather than the PSU-specific value?